.github/workflow-scripts/GenerateCategories.java

```diff
@@ -19,7 +19,7 @@
 public class GenerateCategories {
     
     //Constants
-    static final boolean DEBUG = false;
+    static final boolean DEBUG = true;
```

Comment:
Just want to verify that you always want to collect debug information.

Reply:
Yes, previously enabling debug would corrupt the variable `MATRIX_RESULT` in the build script.  Now debug goes to stderr while the actual output we want to go to MATRIX_RESULT is sent to stdout.